src/app/college/page.tsx

```diff
@@ -3,14 +3,19 @@
 // In this file useSearchParams is a client component.
 'use client'
 
-import jsonData from '../../../data/university_data'
+import jsonData from '../../../data/university_data.json'
```

Comment:
we will ask @bacpactech regarding using the getUniversityList query rather than jsonData.